pkg/drain/config.go

```diff
@@ -0,0 +1,18 @@
+package drain
+
+import (
+	"time"
+)
+
+type NodeDrain struct {
+	Timeout        int `yaml:"timeOut"`
+	WorkerNodeTime int `yaml:"workerNodeTime"`
+}
+
+func (nd *NodeDrain) GetTimeOutDuration() time.Duration {
+	return time.Duration(nd.Timeout) * time.Minute
+}
+
+func (nd *NodeDrain) GetExpectedDrainDuration() time.Duration {
+	return time.Duration(nd.WorkerNodeTime) * time.Minute
+}
```

Comment:
Do you think it would help to have this method be renamed to something else having the reference to `WorkerNodeTime` ? `GetTimeOutDuration` and `Timeout` gives me the idea that the method is about fetching the `Timeout` duration. But `GetExpectedDrainDuration` doesn't directly imply that it would regard worker node time out as specified by `WorkerNodeTime`.

Reply:
Yeah, it does read a bit wrong alright. This has evolved from initially being maintenance config.
If @mrbarge id ok with it, id update it. (probs needs docs updates somewhere too)

Reply:
This will now be:
```yaml
    nodeDrain:
      timeOut: 5
      expectedNodeDrainTime: 8
```